package.json

```diff
@@ -80,13 +79,14 @@
     "@babel/cli": "^7.17.6",
     "@babel/core": "^7.16.0",
     "@babel/eslint-plugin": "^7.17.7",
+    "@babel/plugin-transform-runtime": "^7.17.12",
```

Comment:
Is the config for this missing in babel.config.js? As explained [here](https://babeljs.io/docs/en/babel-plugin-transform-runtime#usage)